packages/integrations/markdoc/README.md

```diff
@@ -249,6 +212,83 @@ export default defineMarkdocConfig({
 })
 ```
 
+### Set the root HTML element
+
+Markdoc wraps documents with an `article` tag by default. This can be changed from the `document` Markdoc node. This accepts an HTML element name or `null` if you prefer to remove the wrapper element:
+
+```js
+// markdoc.config.mjs
+import { defineMarkdocConfig, nodes } from '@astrojs/markdoc/config';
+
+export default defineMarkdocConfig({
+  nodes: {
+    document: {
+      render: null,
+      // Apply Markdoc's defaults for other options
+      ...nodes.document,
+    },
+  },
+})
+```
+
+### Custom Markdoc nodes / elements
+
+You may want to render standard Markdown elements like paragraphs and bolded text with components. For this, you can configure a [Markdoc node][markdoc-nodes]. If a given node receives attributes, they will be available as component props.
+
+This example renders blockquotes with a custom `Quote.astro` component:
+
+```js
+// markdoc.config.mjs
+import { defineMarkdocConfig, nodes } from '@astrojs/markdoc/config';
+import Quote from './src/components/Quote.astro';
+
+export default defineMarkdocConfig({
+  nodes: {
+    heading: {
+      render: Quote,
+      // Apply Markdoc's defaults for other options
+      ...nodes.blockquote,
+    },
+  },
+})
+```
+
+📚 [Find all of Markdoc's built-in nodes and node attributes on their documentation.](https://markdoc.dev/docs/nodes#built-in-nodes)
+
+### Use client-side UI components
+
+Tags and nodes are restricted to Astro components today. To embed client-side UI components, create a wrapper `.astro` file to import your component and apply a `client:` directive manually.
```

Comment:
```suggestion
Tags and nodes are current restricted to `.astro` files. To embed client-side UI components in Markdoc, [use a wrapper `.astro` component that renders a framework component](/en/core-concepts/framework-components/#nesting-framework-components).
```
Would this be an appropriate link here? The "applying a client directive manually" feels a little off. As opposed to....? Don't we always have to write a client directive (manually)?

If this is mostly a reminder to use a client directive, you could also tack on something like:

> renders a framework component]() with your desired `client:` directive.

Reply:
Good edits! Seeing this written out, I want to take out "currently," since we probably _won't_ support this in the future. And yes, good link 👍